tests/test_all_integration.py

```diff
@@ -0,0 +1,335 @@
+import tests.context
+import pytest
+import time
+from multiprocessing import Process
+import os
+import shutil
+
+from core.communication_manager         import CommunicationManager
+from core.runner                        import DMLRunner
+from core.scheduler                     import DMLScheduler
+from core.configuration                 import ConfigurationManager
+from tests.testing_utils                import make_initialize_job, make_model_json
+from tests.testing_utils                import make_serialized_job, serialize_job
+from core.utils.enums                   import RawEventTypes, JobTypes, MessageEventTypes
+from core.utils.keras                   import serialize_weights
+from core.blockchain.blockchain_gateway import BlockchainGateway
+from core.blockchain.blockchain_utils   import setter, TxEnum
+
+
+session_filepaths = set()
+
+config_manager = ConfigurationManager()
+config_manager.bootstrap(
+    config_filepath='tests/artifacts/integration/configuration.ini'
+)
+
+@pytest.fixture
+def mnist_filepath():
+    return 'tests/artifacts/integration/mnist'
+
+@pytest.fixture
+def new_session_event(mnist_filepath):
+    # initialize_job = make_initialize_job(make_model_json())
+    # initialize_job.hyperparams['epochs'] = 10
+    # initialize_job.hyperparams['batch_size'] = 128
+    # initialize_job.hyperparams['split'] = .9
+    # serialized_job = serialize_job(initialize_job)
```

Comment:
If you're not going to use this anymore, you can go ahead and delete this.

Reply:
Done in upcoming commit